native_client/alphabet.h

```diff
@@ -17,29 +18,29 @@ class Alphabet {
   Alphabet(const char *config_file) {
     std::ifstream in(config_file, std::ios::in);
     unsigned int label = 0;
+    space_label_ = -2;
     for (std::string line; std::getline(in, line);) {
       if (line.size() == 2 && line[0] == '\\' && line[1] == '#') {
         line = '#';
       } else if (line[0] == '#') {
         continue;
       }
-      label_to_str_[label] = line;
+      //TODO: we should probably do something more i18n-aware here
```

Comment:
I fear it's going to badly break for any language using something different than a simple space ; for example, non-breaking spaces in french

Reply:
Some of those cases should be fixed as a preprocessing step, replacing non-breaking spaces with normal spaces in the training set, for example. There's likely cases where that approach doesn't solve it, so I agree we should fix this.